src/Microsoft.Health.Dicom.Core/Features/Model/VersionedInstanceIdentifier.cs

```diff
@@ -14,20 +14,8 @@ public class VersionedInstanceIdentifier : InstanceIdentifier
         string seriesInstanceUid,
         string sopInstanceUid,
         long version,
-        PartitionEntry partitionEntry)
-        : base(studyInstanceUid, seriesInstanceUid, sopInstanceUid, partitionEntry)
-    {
-        Version = version;
-    }
-
-    public VersionedInstanceIdentifier(
-        string studyInstanceUid,
-        string seriesInstanceUid,
-        string sopInstanceUid,
-        long version,
-        int partitionKey = DefaultPartition.Key,
-        string partitionName = DefaultPartition.Name)
-        : base(studyInstanceUid, seriesInstanceUid, sopInstanceUid, partitionKey, partitionName)
+        Partition partition = null)
```

Comment:
In which scenario do you want to accept null partition?

Reply:
Is there any repercussion of making this change? we are introducing a new change which wasn't there before